app/helpers/reports/regions_helper.rb

```diff
@@ -4,6 +4,11 @@ def reports_region_facility_path(facility, options = {})
     reports_region_path(facility, options)
   end
 
+  def reports_region_district_path(district, options = {})
```

Comment:
Why a helper for this instead of setting the `report_scope`, like we do for the facility paths?

Reply:
I've got helpers for both facility and facility group now, just haven't switched over to use them everywhere (yet).